web/services/premium_detection.go

```diff
@@ -0,0 +1,72 @@
+package services
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	Premium   = "Premium"
+	Community = "Community"
```

Comment:
I don't think these constants should be public, should they?

Reply:
ok, changed. thanks